debug/src/check-props.js

```diff
@@ -0,0 +1,28 @@
+
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
The whole `preact/debug` import has this check around all code in the entry file. We can safely omit it here :+1:

Reply:
This was cleaned up in 9bcfb14